### ISSUE TITLE
Remove the YamlWire.writeObject(Object o) method

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Wire#511]
Remove the YamlWire.writeObject(Object o) method, instead change the getValueOut().object() method to do the same thing.

Fix  

```
net.openhft.chronicle.wire.TextWireCompatibilityTest
```

**Comment by RobAustin** [comment]
One approach is to set the system.properties to :

```
wire.testAsYaml=true
```

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.23ea21](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.23ea21), [BOM-2.23ea75](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea75)
